package.json

```diff
@@ -55,6 +55,7 @@
     "@chiragrupani/karma-chromium-edge-launcher": "2.1.1",
     "@jest/core": "27.0.4",
     "@jridgewell/doctrine": "3.0.2",
+    "@octokit/graphql": "4.6.4",
```

Comment:
Updates to package.json also need updates to package-lock.json, don't they?

Reply:
this and dedent were already nested dependencies in the lock file.